.github/labeler.yml

```diff
@@ -0,0 +1,11 @@
+Documentation:
+  - changed-files:
+      - any-glob-to-any-file: ['docs/*', '.readthedocs.yml', 'logos/*', 'README.md']
+
+Internal:
+  - changed-files:
+      - any-glob-to-any-file: ['.pre-commit-config.yaml', '.readthedocs.yml', 'codecov.yml', '.gitignore', '.github/*', 'nigsp/_version.py']
```

Comment:
Debatable, but what if we just set a rule encompassing all hidden (.*) files in the root for "Internal"? They have been treated in the same way so far (We don't make enough granular distinctions with the labels).
Or do you want to set up another label/category?

```suggestion
      - any-glob-to-any-file: ['.*', 'codecov.yml', 'setup.cfg', 'setup.py', 'versioneer.py', '.github/*', '.circleci/*', 'nigsp/_version.py']
```

Reply:
Sounds good to me